odoo_module_migrate/migration_scripts/text_replaces/migrate_120_130/digits_precision.yaml

```
@@ -0,0 +1,4 @@
+.py:
+  import\sodoo\.addons\.decimal_precision\sas\sdp.*\n:
```

no need to check import name
```suggestion
  import\sodoo\.addons\.decimal_precision\b.*\n:
```